Lib/fontmake/font_project.py

```diff
@@ -1179,7 +1162,7 @@ def buildTables(self):
             stdout, stderr = process.communicate()
             retcode = process.poll()
 
-            report = tostr(stdout + (b"\n" + stderr if stderr else b""))
+            report = str(stdout + (b"\n" + stderr if stderr else b""))
```

Comment:
don't call `str()`, but do the decoding to 'ascii', since the output from subprocess is always bytes.

```python
(stdout + (b"\n" + stderr if stderr else b"")).decode("ascii")
```

Reply:
if you were to do `str(b"a")` you'd get the bytes repr "b'a'" back, which is not what you want.

Reply:
or if you want to keep calling `str()` you need to pass the `encoding="ascii"` keyword argument so that it in turn calls the decode method for you.